src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,149 @@
 import React from 'react';
 import './TodoList.scss';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+class TodoList extends React.PureComponent {
+  state = {
+    search: '',
+    todoStatus: 'all',
+    isShuffle: false,
+  }
+
+  searchHandler = (event) => {
+    this.setState({
+      search: event.target.value,
+    });
+  }
+
+  getTodosForShow = (todos) => {
+    if (!todos) {
+      return [];
+    }
+
+    const { search } = this.state;
+    const searchToLower = search.toLowerCase();
+
+    const todosWithSearchFilter = todos.filter(({ title }) => (
+      title && title.toLowerCase().includes(searchToLower)));
```

Comment:
Все скобки которые открылись в прошлой строке и не закрылись должны быть закрыты в следующей, с тем же отступом что и в строке открытия
```suggestion
    const todosWithSearchFilter = todos.filter(({ title }) => (
      title && title.toLowerCase().includes(searchToLower)
    ));
```